src/Components/MSBuild.fs

```diff
@@ -220,6 +220,20 @@ module MSBuild =
         let host = tryGetRightHost' path
         invokeMSBuild path target (Some host)
 
+    let restoreMailBox = MailboxProcessor.Start(fun inbox-> 
```

Comment:
MailboxProcessor.Start now work in fable too?

Reply:
Yes, there's [limited support for MailboxProcessor](http://fable.io/docs/compatibility.html#fsharp-core) (check the caveats)